### PR TITLE
[DOC] update FSL preprocess

### PR DESCRIPTION
**Pull request by neuroumbrage**
@satra Does this work?

### CODE REVIEW

**Comment by satra** [comment]
yes - much cleaner. thank you.

File: nipype/interfaces/fsl/preprocess.py [hunk 1]
```diff
@@ -594,6 +595,16 @@ class ApplyXFM(FLIRT):
...
+class ApplyXfm(ApplyXFM):
+    """
+    .. deprecated:: 0.12.1
+       Use :py:class:`nipype.interfaces.fsl.ApplyXFM` instead
+    """
+    def __init__(self, **inputs):
+        super(ApplyXfm, self).__init__(**inputs)
+        warn(('This interface has been renamed since 0.12.1, please use '
+              'nipype.interfaces.fsl.ApplyXFM'),
+             UserWarning)
```
Comment (effigies) [inline nipype/interfaces/fsl/preprocess.py]: This was removed in #1866. Looks like an accidental re-add.

File: nipype/interfaces/fsl/preprocess.py [hunk 2]
```diff
@@ -908,6 +923,9 @@ class FNIRTOutputSpec(TraitedSpec):
 class FNIRT(FSLCommand):
     """Use FSL FNIRT for non-linear registration.
 
+    For complete details, see the `MCFLIRT Documentation.
```
Comment (effigies) [inline nipype/interfaces/fsl/preprocess.py]: `MCFLIRT` => `FNIRT`

File: nipype/interfaces/fsl/preprocess.py [hunk 3]
```diff
@@ -1208,6 +1226,9 @@ class SUSANOutputSpec(TraitedSpec):
 class SUSAN(FSLCommand):
     """ use FSL SUSAN to perform smoothing
 
+    For complete details, see the `MCFLIRT Documentation.
```
Comment (effigies) [inline nipype/interfaces/fsl/preprocess.py]: Likewise, `MCFLIRT` => `SUSAN`.

**Pull request merged**